internal/redfishwrapper/task.go

```diff
@@ -97,6 +95,8 @@ func (c *Client) TaskStateActive(state constants.TaskState) (bool, error) {
 		return true, nil
 	case constants.Complete, constants.Failed:
 		return false, nil
+	case constants.PowerCycleHost, constants.Unknown:
+		fallthrough
```

Comment:
For what is this optimizing? Why is this needed if it's just going to fallthrough to the default? Won't the default case be called regardless of whether this case exists or not?

Reply:
Oh, is this to satisfy a linting complaint?

Reply:
`constants.PowerCycleHost` and `constants.Unknown` should not be part of the switch case since they are not Redfish task states,
could we have this case removed?

http://redfish.dmtf.org/schemas/DSP2046_2019.4.html#redfish.dmtf.org/schemas/v1/Task.json|details|TaskState